src/comlink.ts

```diff
@@ -584,9 +584,20 @@ function myFlat<T>(arr: (T | T[])[]): T[] {
   return Array.prototype.concat.apply([], arr);
 }
 
+/** Get the first item of a tuple */
+function firstItem<O, T>(v: [O, T]): O {
```

Comment:
Nit:  `unknown` would do as well since the second type is not used in the return  type
```suggestion
function firstItem<T>(v: [T, unknown]): T {
```